docker/Dockerfile.dev

```diff
@@ -0,0 +1,53 @@
+FROM python:3.8-slim as base
+
+# When building locally, these should be set to your UID/GID.  That way, any
+# files written to the $PWD mount will be owned by you.  This is not
+# necessary (or wanted) when building in Concourse.
+ARG UID=2000
+ARG GID=2000
+ARG USER=app
```

Comment:
You would have to change these locally when you clone the repo, but they stay as is for deploying through Concourse?